.travis.yml

```diff
@@ -12,7 +12,7 @@ cache:
 
 install:
     - pip install -U pip
-    - pip install torch==1.4.0+cpu torchvision==0.5.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
+    - pip install torch==1.5.0+cpu torchvision==0.6.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
I guess you'll want to move into the torch backend only, right?

Reply:
Thanks, I should do so.